### ISSUE TITLE
”SverigesRadio will not be installed”

### ISSUE THREAD

**Issue by envackervirvel** [ljungberg/sverigesradio#1]
Hi!
I get following error message when trying to install the plugin:
 [21-12-07 11:11:57.0966] Slim::Utils::PluginDownloader::_installDownload (210) digest does not match /usr/local/slimserver/Cache/DownloadedPlugins/SverigesRadio.zip - SverigesRadio will not be installed: expected 439c3e1f5bf94c58463698aa619a7dec52e45cf5, got 3a301f33e48101954a81bcbf73cd1539d13bcb9d
 
System info:
Logitech Media Server Version: 8.1.0 - 1608700893 @ Wed Dec 23 12:16:44 CET 2020
Värdnamn: piCorePlayer
Serverns IP-adress: 192.xxx.xxx.xxx
HTTP-serverns portnummer: 9000
Operativsystem: piCore - SV - utf8
Plattformsarkitektur: armv8l-linux
Perl-version: 5.28.2 - arm-linux-gnueabihf-thread-multi-64int
Audio::Scan: 1.02
IO::Socket::SSL: 2.066
Databasversion: DBD::SQLite 1.58 (sqlite 3.22.0)

/Fredrik

**Comment by ljungberg** [comment]
Thanks!
When I updated it 5 months ago I forgot to replace the SHA checksum.
It is now fixed.
Try to remove and then add the repository (http://ljungberg.github.io/sverigesradio/repo.xml) in LMS again

**Comment by envackervirvel** [comment]
Oki.
I removes the repository and restarted LMS. Then added it again and restarted and got this (no idea what this error is I’m afraid):

[21-12-09 22:05:59.7814] main::init (388) Starting Logitech Media Server (v8.1.0, 1608700893, Wed Dec 23 12:16:44 CET 2020) perl 5.028002 - arm-linux-gnueabihf-thread-multi-64int
[21-12-09 22:06:01.3922] Slim::Utils::Misc::msg (1266) Warning: [22:06:01.3915] DBIx::Class::ResultSet::update_or_create(): Query returned more than one row.  SQL that returns multiple rows is DEPRECATED for ->find and ->single at /usr/local/slimserver/Slim/Schema.pm line 1899
[21-12-09 22:06:02.6655] Slim::bootstrap::tryModuleLoad (286) Warning: Module [Plugins::SverigesRadio::Plugin] failed to load:
Experimental push on scalar is now forbidden at /usr/local/slimserver/Cache/InstalledPlugins/Plugins/SverigesRadio/Plugin.pm line 151, near "$channel;"
Compilation failed in require at (eval 938) line 1.
BEGIN failed--compilation aborted at (eval 938) line 1.

[21-12-09 22:06:02.6680] Slim::Utils::PluginManager::load (323) Error: Couldn't load Plugins::SverigesRadio::Plugin

**Comment by ljungberg** [comment]
The error you get now is the one I fixed 5 months ago...

I somehow forgot to include the actual fix in the zip file 5 months ago when I updated :(

I have changed it now so try to remove the pugin in LMS and add / remove the repo again.

**Comment by envackervirvel** [comment]
Hm, maybe I'm doing something wrong, but I removed the repo -> restarted LMS. -> added the repo again - > installed the plugin. But I get this error message:
[21-12-11 10:21:17.2150] Slim::Utils::PluginDownloader::_installDownload (210) digest does not match /usr/local/slimserver/Cache/DownloadedPlugins/SverigesRadio.zip - SverigesRadio will not be installed: expected 3a301f33e48101954a81bcbf73cd1539d13bcb9d, got 31517a3b07cbba88a26fd2e42dc456f82a0dee54

**Comment by ljungberg** [comment]
No, you are not doing anything wrong it was me that forgot to update the md5sum after updating the zip file. I have updated the md5sum now.
3rd times the charm?

**Comment by envackervirvel** [comment]
Seems to work like a charm, now it is installed and running. Thank you!
